File: Node/ConsolePrint/src.js
console.time("Execution time: ");

let ti = 5000;
let tj = 20;
let p = ti * tj;
let s = 0;
for (let i = 0; i <= ti; i++)
{
    for (let j = 0; j <= tj; j++)
    {
        console.log(s * 100 / p);
    }
    s = s + tj;
}

console.timeEnd("Execution time: ");

File: Python/ConsolePrint/src.py


import time
start_time = time.time()

ti = 5000
tj = 20
p = ti * tj
s = 0

for i in range(0,ti):
    for j in range(0, tj):
        r = s * 100 / p
        print(r)
    s = s + tj

print("Execution time: %s" % (time.time() - start_time))